erpnext/stock/doctype/stock_entry/stock_entry.py

```diff
@@ -1398,6 +1400,15 @@ def update_package_tag(self):
 						frappe.throw(_("Do not assign new item code to an existing package tag."))
 
 
+	def update_package_tag_transaction_status(self):
+		for item in self.items:
+			if not item.package_tag:
+				continue
+
+			exists = 1 if frappe.db.exists("Stock Ledger Entry", {"package_tag": item.package_tag}) else 0
```

Comment:
@hrwX always also check if the tag is lost or destroyed, if yes. Always set this to true.

In case the package tag is marked destroyed first and later a stock entry that had the package tag is cancelled, this will make the package tag available

Reply:
@neilLasrado added a check for checking whether the package tag is set to lost/destroyed